### PR TITLE
Multi-chat conversation view

### PR DESCRIPTION
**Pull request by LUMacKenn**
I've updated the single conversation view and incorporated both the conversation list view and single conversation view into the live app. Neither are behind feature flags, the old views are not being used anymore (will delete old components later).

Known issues:
- The avatars on the list of conversations will currently only display blank circles when the last message was sent by a care team member or a patient without a name, I'll get to this in a followup PR
- Lots of this code was taken and modified from kariko so there may be some artifacts in the code of support for SMS in addition to chat. I have removed it in some places where I saw it but will continue to do this going forward, as there is no need for SMS in this patient-facing app

[sc-7052]
[sc-7053]

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,18 +5,20 @@ import { ThemeProvider } from "@mui/material/styles";
 import CssBaseline from "@mui/material/CssBaseline";
 import { SWRConfig } from "swr";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
-// Providers
```
Comment (lawyu89) [inline src/App.js:8]: I feel like it was more appropriate to group the AppAuthenticator as a provider rather than a page.

File: src/features/conversations/components/ConversationContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import MessageSendBar from "./MessageSendBar";
+import { useBarnardConversation } from "../hooks/useBarnardConversation.hook";
+import { useSendMessageByConversationSid } from "../hooks/useSendMessage.hook";
+import { useTwilioConversation } from "../hooks/useTwilioConversation.hook";
+
+const ConversationContainer = () => {
+  const { conversationId } = useParams<{
+    id: string;
+    conversationId: string;
+  }>();
```
Comment (lawyu89) [inline src/features/conversations/components/ConversationContainer.tsx:17]:
```suggestion
  const { conversationId } = useParams<{
    conversationId: string;
  }>();
```

File: src/features/conversations/components/ConversationMessagesLoader.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        overflowY: "hidden",
+      }}
+    >
+      <MessageSkeleton />
+      <MessageSkeleton />
+      <MessageSkeleton />
+      <MessageSkeleton />
+      <MessageSkeleton />
+      <MessageSkeleton />
+      <MessageSkeleton />
```
Comment (lawyu89) [inline src/features/conversations/components/ConversationMessagesLoader.tsx:25]: Maybe we can shorten this to something like this?
```suggestion
      {[...Array(7)].map((_e, i) => <MessageSkeleton key={i} />)}
```

File: src/features/conversations/components/ConversationsList.tsx [hunk 4]
```diff
@@ -2,10 +2,7 @@ import { Conversation, Message } from "@twilio/conversations";
 import { useEffect, useState, Fragment } from "react";
 import Box from "@mui/material/Box";
 
-import {
-  BarnardConversation,
-  ConversationTypes,
-} from "../../../models/conversations/BarnardConversation.types";
+import { BarnardConversation } from "../../../models/conversations/BarnardConversation.types";
```
Comment (lawyu89) [inline src/features/conversations/components/ConversationsList.tsx:5]:
```suggestion
import { BarnardConversation } from "models/conversations/BarnardConversation.types";
```

**Review by lawyu89 — APPROVED** [review]
Looks really good! Approved!

File: src/App.js (hunk 1 above)
Comment (LUMacKenn) [reply src/App.js:8]: Thanks, didn't realize I did this, must have been leftover from a merge
